pkg/reconciler/pipelinerun/controller.go

```diff
@@ -85,6 +90,10 @@ func NewController(opts *pipeline.Options, clock clock.Clock) func(context.Conte
 			FilterFunc: controller.FilterController(&v1beta1.PipelineRun{}),
 			Handler:    controller.HandleAll(impl.EnqueueControllerOf),
 		})
+		resourceRequestInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
```

Comment:
Yes you're absolutely right, and this is why the integration tests are all failing badly.

The only workaround I can think of right now is to put this initialization behind an environment variable that the user must explicitly enable.

Reply:
OK, unfortunately it doesn't look like I can optionally start informers like that, I'll have to think of something else.

Reply:
I saw the slack conversation you opened on pipeline-dev; I agree with Matt Moor that it would be OK to import the ResourceRequest CRD as part of the Tekton pipelines config.

Reply:
I've brought the CRD over into pipelines. One unfortunate downside is that this means the CRD is duplicated both in resolution and here. I tried vendoring the CRD yaml in resolution but `go mod` doesn't pull anything except go code into pipelines.

Reply:
ack -- that's too bad but it doesn't sound like there's a good way around this duplication.

Reply:
I've removed the CRD from resolution. Now the definition in Pipelines is the only one and Tekton Resolution will lean on that rather than the other way round.